.github/workflows/build.yml

```diff
@@ -0,0 +1,24 @@
+name: Build
+
+on:
+  push:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-18.04
```

Comment:
```suggestion
    runs-on: ubuntu-latest
```

Reply:
Thanks. Done.

Reply:
The `ubuntu-latest` maybe change when new ubuntu released, and it causes some GitHub actions failed some months ago when GitHub changed `ubuntu-latest` to `ubuntu-20.0.4`, https://github.com/actions/virtual-environments/issues/1816.